### PR TITLE
Use elseif instead else if.

### PR DESCRIPTION
**Pull request by realityking**
Main reason is that it is much easier to parse when applying other style checks but it's also more consistent and as an added bonus there is a very small speed increase involved (probably < 1%).

### CODE REVIEW

File: build/phpcs/Joomla/Sniffs/ControlStructures/ElseIfDeclarationSniff.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        $phpcsFile->addError($error, $stackPtr, 'NotAllowed');
+      }
+
+    }//end process()
+
+
+}//end class
+
+
+?>
```
Comment (chdemko) [inline build/phpcs/Joomla/Sniffs/ControlStructures/ElseIfDeclarationSniff.php]: Why did you add a ?> at the end of the file?

**Comment by realityking** [comment]
Good point, fixed.

**Comment by LouisLandry** [comment]
Fantastic!

**Comment by elkuku** [comment]
I **love** GitHub `:)`
